__tests__/app.test.js

```diff
@@ -214,3 +214,60 @@ describe('POST /api/articles/:article_id/comments', () => {
             });
     });
 });
+
+describe('PATCH /api/articles/:article_id', () => {
+    test('should update the article votes and respond with the updated article', () => {
+        return request(app)
+            .patch('/api/articles/1')
+            .send({ inc_votes: 1 })
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.article).toEqual(
+                    expect.objectContaining({
+                        article_id: 1,
+                        votes: expect.any(Number),
```

Comment:
Have a look at how many votes the article contains in your seed data so you can expect the actual number after the change otherwise how do you know that its changing